apps/functions-subscription/src/adapters/azure/cosmosdb/subscription.ts

```diff
@@ -24,6 +30,64 @@ export const makeSubscriptionCosmosContainer = (
         ),
         TE.flatMapEither(decodeFromItem(SubscriptionCodec)),
       ),
+    listByTrialId: (trialId, pageSize, maximumId?, minimumId?) =>
```

Comment:
**issue(blocking)**: should we list the subscriptions for every trial or should we just get a subscription of a trial?

During refinement I thought we said that we need to allow the `TrialSupport` user to get details of a user's subscription, but I may be wrong. cc @AleDore 

In the second case (list subscriptions) then you should pay attention to the query, because the `subscription` container has the `id` property as partition key.

Reply:
we have clarified the User Story related to this activity and the goal is to give the Support user the ability to see the state of single a user in a trial.
there is no need to list the subscriptions by trial.

cc @PascalDR